.gitignore

```diff
@@ -46,3 +46,19 @@ lerna-debug.log*
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+
+# Yarn
+.yarn/*
+!.yarn/.pnp.*
+!.yarn/cache
+!.yarn/unplugged
+!.yarn/build-state.yml
+!.yarn/install-state.gz
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
+.pnp.*
+.yarnrc
+yarn.lock
```

Comment:
```suggestion

```

Ok. yarn.lock is required. Please don’t ignore this

Reply:
got it

Reply:
https://yarnpkg.com/getting-started/qa/#which-files-should-be-gitignored

Reply:
which one that you recommend the most?

Reply:
Just follow the documentation

Reply:
I dont familiar with it 😅